nari/ext/act/utils.py

```diff
@@ -11,34 +11,29 @@
 DEFAULT_DATE_FORMAT: str = '%Y-%m-%dT%H:%M:%S.%f%z'
 
 
-# pylint: disable=invalid-name
-class ActLogChecksumType(Enum):
-    """List of hashsum algorithms used by different ACT versions"""
-    MD5 = "md5"
-    SHA256 = "sha256"
-# pylint: enable=invalid-name
+try:
+    from nari_act_rust.utils import date_from_cs_string, validate_checksum_internal
+except ImportError:
+    def date_from_cs_string(datestr: str) -> Timestamp:
+        """Look, this is dirty. This is wrong. Please someone find a better way to do this."""
+        return int(datetime.strptime(f'{datestr[:26]}{datestr[-6:]}', DEFAULT_DATE_FORMAT).timestamp() * 1000)
 
+    def validate_checksum_internal(line: str, index: int) -> bool:
```

Comment:
Shouldn't this be `validate_checksum_py`? Please handle these consistently with the established pattern.

Reply:
Would be `validate_checksum_internal_py` due to it having to switch between the internal function of either python or rust